app/indexers/descriptive_metadata_indexer.rb

```diff
@@ -199,5 +207,20 @@ def flat_event(event)
   def flat_value(value)
     value.parallelValue.presence || value.groupedValue.presence || value.structuredValue.presence || Array(value)
   end
+
+  # @param [Cocina::Models::Contributor] contributor to check
+  # @return [Boolean] true unless the contributor has a citation status of false
+  def cited?(contributor)
+    contributor.note.none? { |note| note.type == 'citation status' && note.value == 'false' }
+  end
+
+  # @param [Cocina::Models::Contributor] contributor to check
+  # @return [String, nil] orcid id including host if present
+  def orcidid(contributor)
+    identifier = contributor.identifier.find { |id| id.type == 'ORCID' }
+    return unless identifier
+
+    URI.join(identifier.source.uri, identifier.value).to_s
+  end
 end
```

Comment:
NOTE: these methods to search for ORCIDs were lifted from https://github.com/sul-dlss/orcid_client/blob/main/lib/sul_orcid_client/cocina_support.rb

While it would be nice to not copy and paste them here, that would require either (1) adding the sul-orcid-client gem has a dependency here or (2) moving these methods somewhere else that is available to both this app and the sul-orcid-client gem.  Currently opting for neither.